openpype/hosts/houdini/api/lib.py

```diff
@@ -121,104 +121,73 @@ def get_id_required_nodes():
     return list(nodes)
 
 
-def get_export_parameter(node):
-    """Return the export output parameter of the given node
-
-    Example:
-        root = hou.node("/obj")
-        my_alembic_node = root.createNode("alembic")
-        get_output_parameter(my_alembic_node)
-        # Result: "output"
-
-    Args:
-        node(hou.Node): node instance
-
-    Returns:
-        hou.Parm
-
-    """
-    node_type = node.type().description()
-
-    # Ensures the proper Take is selected for each ROP to retrieve the correct
-    # ifd
-    try:
-        rop_take = hou.takes.findTake(node.parm("take").eval())
-        if rop_take is not None:
-            hou.takes.setCurrentTake(rop_take)
-    except AttributeError:
-        # hou object doesn't always have the 'takes' attribute
-        pass
-
-    if node_type == "Mantra" and node.parm("soho_outputmode").eval():
-        return node.parm("soho_diskfile")
-    elif node_type == "Alfred":
-        return node.parm("alf_diskfile")
-    elif (node_type == "RenderMan" or node_type == "RenderMan RIS"):
-        pre_ris22 = node.parm("rib_outputmode") and \
-            node.parm("rib_outputmode").eval()
-        ris22 = node.parm("diskfile") and node.parm("diskfile").eval()
-        if pre_ris22 or ris22:
-            return node.parm("soho_diskfile")
-    elif node_type == "Redshift" and node.parm("RS_archive_enable").eval():
-        return node.parm("RS_archive_file")
-    elif node_type == "Wedge" and node.parm("driver").eval():
-        return get_export_parameter(node.node(node.parm("driver").eval()))
-    elif node_type == "Arnold":
-        return node.parm("ar_ass_file")
-    elif node_type == "Alembic" and node.parm("use_sop_path").eval():
-        return node.parm("sop_path")
-    elif node_type == "Shotgun Mantra" and node.parm("soho_outputmode").eval():
-        return node.parm("sgtk_soho_diskfile")
-    elif node_type == "Shotgun Alembic" and node.parm("use_sop_path").eval():
-        return node.parm("sop_path")
-    elif node.type().nameWithCategory() == "Driver/vray_renderer":
-        return node.parm("render_export_filepath")
-
-    raise TypeError("Node type '%s' not supported" % node_type)
-
-
 def get_output_parameter(node):
     """Return the render output parameter of the given node
 
     Example:
         root = hou.node("/obj")
         my_alembic_node = root.createNode("alembic")
         get_output_parameter(my_alembic_node)
-        # Result: "output"
+        # Result: "filename"
 
     Args:
         node(hou.Node): node instance
 
     Returns:
         hou.Parm
 
+    Note 1:
```

Comment:
Notes should be before Args and Returns.
```
    Notes:
        I'm using node.type().name() to get on par with the creators,
            Because the return value of `node.type().name()` is the same
            string value used in creators.
            e.g. instance_data.update({"node_type": "alembic"})

        Rop nodes in different network categories have
            the same output parameter. So, I took that into
            consideration as a hint for future development.
```